src/test/resources/gradle/cucumber/simpleGet.feature

```diff
@@ -6,3 +6,4 @@ Feature: Simple GET
     When I send method "GET" for "/simple_get" to host at the specified port
     Then I should receive a response with version "HTTP/1.1"
     And Status code 200
+    And Body ""
```

Comment:
Should the body be `""` or `null`? Same comment for `simpleHead.feature` line 9

Reply:
@avni510 I could not find gherkin support for any kind of null value. I did not know if there was a better way to assert that the body was empty. Do you know of any?

Reply:
Can you do something like `assert actualBody.equals(null)?`

Reply:
Although I'm not sure if that's the conventional way to handle null values. Damon might know